x-pack/elastic-agent/pkg/agent/cmd/enroll.go

```diff
@@ -59,6 +60,8 @@ func addEnrollFlags(cmd *cobra.Command) {
 	cmd.Flags().Uint16P("fleet-server-port", "", 0, "Fleet Server HTTP binding port (overrides the policy)")
 	cmd.Flags().StringP("fleet-server-cert", "", "", "Certificate to use for exposed Fleet Server HTTPS endpoint")
 	cmd.Flags().StringP("fleet-server-cert-key", "", "", "Private key to use for exposed Fleet Server HTTPS endpoint")
+	cmd.Flags().StringSliceP("headers", "", []string{}, "App auth token used for elasticsearch")
```

Comment:
I would name this `header` instead of `headers`. Being that its a multiple parameter, then it cleaner:

`--header headera=1 --header headerb=2`